docs/getting_started/index.rst

```diff
@@ -1,115 +1,164 @@
+General Information about Mantl with Ansible
+============================================
 
-Provisioning Cloud Hosts
-------------------------
+The Mantl project uses Ansible to bring up
+nodes and clusters. This generally means that you need three things:
+
+1. Hosts to use as the base for your cluster
+2. An `inventory file`_ with the hosts you want to be modified. Mantl includes
+   an Ansible inventory for terraformed hosts, `terraform.py`_.
+3. A playbook matching Ansible roles to hosts. Mantl organizes its components
+   in `sample.yml`_, which we recommend copying to ``mantl.yml`` for the
+   possibility of later customization. You can read more about `playbooks`_ in
+   the `Ansible docs`_.
+
+Preparing to provision Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 
 The playbooks and roles in this project will work on whatever provider
 (or metal) you care to spin up, as long as it can run CentOS 7 or
-equivalent. However, here are some guides to get started on common
-platforms:
+equivalent.
 
-.. toctree::
-   :maxdepth: 1
+Your hosts will have to be accessible with your SSH key. If you're unfamiliar
+with SSH keys, please read `this blog post
+<https://www.digitalocean.com/community/tutorials/how-to-set-up-ssh-keys--2>`_.
 
-   openstack.rst
-   gce.rst
-   aws.rst
-   digitalocean.rst
-   vsphere.rst
-   softlayer.rst
+There are several preparatory steps to provisioning the cloud hosts that are
+common to all providers:
 
-Setting up DNS
---------------
+Step 1: Copy .tf file
+---------------------
 
-You can set up your DNS records with Terraform:
+You will also need to copy the .tf file of the platform you are using from
+`mantl/terraform/`_ to the root of the project. For example,
+``mantl/terraform/openstack-modules.sample.tf`` will need to be copied to
+``mantl/openstack-module-sample.tf``. The variables in the copied .tf file will
+need to be changed to your configuration.
 
-.. toctree::
-   :maxdepth: 2
+.. note::
 
-   dns.rst
+    More than one .tf file in the mantl directory will lead to errors upon
+    deployment. If you work with more than one provider, extra .tf files will
+    need to be renamed or moved.
 
+Step 2: Run security-setup
+--------------------------
+
+Running the ``security-setup`` script in the root directory will create and set
+up passwords, authentication, and certificates. For more information, see the
+`security-setup`_ documentation.
 
-Setting up Authentication and Authorization
--------------------------------------------
+Step 3: Set up DNS records
+--------------------------
 
-Before you begin, you'll want to run the ``security-setup`` script in the
-root directory. This will create and set passwords, authentication, and
-certificates. For more information, see the :doc:`security-setup
-<../security/security_setup>` documentation.
+You can set up your DNS records with Terraform: `dns.rst`_
+
+Provisioning Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
Are these merge markers or headers? They looks similar. Elsewhere in the docs, we've used `'''''''''` to indicate third-level headers (although I may be misremembering, check and go with the Sphinx project's common practice if we don't have any)